<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>alert</class>
 <widget class="QDialog" name="alert">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>971</width>
    <height>712</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>851</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Alertes de Ressources</string>
   </property>
  </widget>
  <widget class="QPushButton" name="respondButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>560</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Répondre </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="searchEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>441</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	background-color: #F8F8F8;
	    color: blue;
    border: 1px solid white;
    border-radius: 10px;
    padding: 7px; 
    selection-background-color: #6dae84;
    font-size: 18px; 
}

QLineEdit:focus {
    border: 1px solid #FFFFFF;
}
QlineEdit:hover{
border: 1px solid #6dae84;
}</string>
   </property>
   <property name="placeholderText">
    <string extracomment="nom de l'utilisateur">Rechercher</string>
   </property>
  </widget>
  <widget class="QPushButton" name="refreshButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>560</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Rafraîchir </string>
   </property>
  </widget>
  <widget class="QPushButton" name="exportButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>560</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Exporter </string>
   </property>
  </widget>
  <widget class="QPushButton" name="checkStockButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>660</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Check Stock</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>240</y>
     <width>27</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
